airgun/views/common.py

```diff
@@ -385,12 +384,10 @@ class TemplateEditor(View):
         editor = View.nested(TemplateEditor)
 
     """
-    ROOT = ".//div[@class='editor-container']"
-    rendering_options = ToggleRadioGroup(".//div[@class='btn-group']")
-    import_template = Text(".//a[normalize-space(.)='Import']")
-    fullscreen = Text(".//a[normalize-space(.)='Fullscreen']")
-    syntax_type = Select(id='mode')
-    key_binding = Select(id='keybinding')
+    ROOT = ".//div[@id='editor-container']"
+    rendering_options = ItemsList(".//div[contains(@class,'navbar-editor')]/ul")
+    import_template = Button(".//button[@id='import-btn']")
+    fullscreen = Button(".//button[@id='fullscreen-btn']")
```

Comment:
Why `Button(".//button[@id='fullscreen-btn']")` and not `Button(id='fullscreen-btn')`? Second one doesn't work?